lib/cvodes_2.9.0/src/nvec_ser/nvector_serial.c

```diff
@@ -270,6 +270,7 @@ void N_VPrint_Serial(N_Vector x)
   long int i, N;
   realtype *xd;
 
+#ifndef NO_FPRINTF_OUTPUT
```

Comment:
This `ifndef` should go above the variables or you're going to trigger a bunch of unused variable warnings.

This seems to be a pattern in all of these, but I'm only mentioning it once here.